react/components/withImage.js

```diff
@@ -7,10 +7,10 @@ function getDisplayName(Component) {
 /**
  * HOC used for lazily load an image in the 'images' folder right above
  * this component.
- * 
- * Used for loading the logos for the Footer 
- * 
- * @param {Function} getImageFilename 
+ *
+ * Used for loading the logos for the Footer
+ *
+ * @param {Function} getImageFilename
  */
 export default function withImage(getImageFilename) {
```

Comment:
By default we are using arrow functions, unless when the arrow notation is not possible. Change this to keep the pattern.